src/windows_utils.rs

```diff
@@ -240,6 +195,6 @@ pub fn unregister_right_click() {
 
 pub fn windows_fix_dpi() {
     unsafe {
-        SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
+        SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2).unwrap();
```

Comment:
I think we could use `expect()` instead of unwrap here, with a string describing the error. It's easier for us to know what went wrong that way, since we don't always have the callstack available when users report errors to us.